README.md

```diff
@@ -64,7 +64,6 @@
 - [ ] **[The user can create a new course by himself]**
```

Comment:
- [ ] When in the mobile view, I can't find a way to close the Navbar when selecting a section.  Please either add a "X" to close the navbar, or make it close when a user selects an option. 👍 

_tip:  The "enroll" section seems to close the navbar automatically, but not the others._

That way, your UX will be improved.